web/styles/settings.css

```diff
@@ -1001,6 +1001,8 @@ input[type="checkbox"] {
 
     #current_bot_avatar_image {
         margin: 5px 0 8px;
+        width: 100px;
```

Comment:
Where are the 100px by 100px values coming from? Is that used elsewhere in settings UI for avatars?

Seems like we should also have a quick design discussion about the space around the avatar. It looks odd somehow to havew it closer to the button than to the label. I'm not sure.

Reply:
when user uploads avatar 100px by 100px values are used.
![image](https://github.com/user-attachments/assets/fcfef894-4270-4553-a6ac-bf7a24cd0cc7)

Reply:
Can you use a CSS variable for these such that both values are declared as the same?